walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -14,10 +14,121 @@
 
 package com.google.sps;
 
+import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.Collection;
+import java.util.List;
 
+/** 
+ * Returns a collection of time ranges in which the meeting could fit 
+ * Time complexity is O(m*n^2) in which m is the number of Events and n is the number of Attendees
```

Comment:
Optional: I'd suggest mentioning time complexity only if you think that this could be done in a more efficient case, and if that's the case, I'd suggest using a TODO (e.g. "// TODO: This method is O(...) because ...., this could be improved to be more efficient by ...."). I'd also include it if you think it could be a warning to future users (e.g. "Don't use this code synchronously, or for large numbers of calls because it is..."), but in general I don't think you have to describe the time complexity of the method.

Reply:
Unless of course the tutorial you are following says to do this, in which case ignore what I said :)

Reply:
Yes, it was in the tutorial, thank you!